leadtutes.tex

```diff
@@ -52,5 +52,44 @@ \subsubsection{Alert Production \label{sect:ap}}
 
 \subsection {Caltech IPAC}\label{sect:ipac}
 \subsection {SLAC}\label{sect:slac}
+SLAC hosts the DM Software Architect (\secref{role:softarc}) and the Science Data Archive and Data Access
+Services group described below.
+
+\subsubsection{Science Data Archive and Data Access Services \label{sect:dax}}
+
+The Science Data Archive and Data Access Services (DAX) group has the following major areas of activity
+within DM:
+
+\begin{itemize}
+
+  \item{Provides software to support ingestion, indexing, query, and administration of DM catalog and image
+  data products, data provenance, and other associated metadata within the LSST Data Acess Centers;}
+
+  \item{Provides implementations of data access services (including IVOA services), as well as associated
+  client libraries, to be hosted within the LSST Data Access Centers, which facilitate interraction between
+  LSST data products and tools provided by both other parts of the LSST project and by the astronomical
+  research community at large;}
+
+  \item{Provides a Python framework (the "Data Butler"), used by the LSST science pipelines, to facilitate
+  abstract interaction between in-memory Python objects and generic persisted archives of those objects;}
```

Comment:
"interaction" sounds extremely generic -- shouldn't this be specifically about I/O?